cpp/examples/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
-add_example(wavelets LIBRARIES sopt LABELS wavelets)
-add_example(sara LIBRARIES sopt LABELS wavelets)
-add_example(positive_quadrant_projection LIBRARIES sopt LABELS proximals)
-add_example(l1_norm LIBRARIES sopt LABELS proximals)
-add_example(soft_threshhold LIBRARIES sopt LABELS proximals)
 add_example(conjugate_gradient LIBRARIES sopt)
+add_example(l1_norm LIBRARIES sopt LABELS proximals)
 add_example(l1_proximal LIBRARIES sopt)
+add_example(positive_quadrant_projection LIBRARIES sopt LABELS proximals)
 add_example(power_method LIBRARIES sopt)
+add_example(sara LIBRARIES sopt LABELS wavelets)
+add_example(soft_threshhold LIBRARIES sopt LABELS proximals)
+add_example(wavelets LIBRARIES sopt LABELS wavelets)
 
-add_subdirectory(sdmm)
-add_subdirectory(proximal_admm)
-add_subdirectory(primal_dual)
 add_subdirectory(forward_backward)
+add_subdirectory(primal_dual)
+add_subdirectory(proximal_admm)
+add_subdirectory(sdmm)
```

Comment:
Is this just GitHub being weird, or did you mean to change something?

Reply:
This was just me rearranging the file order so it is alphabetical to make it easier to find the right script in my file explorer